### ISSUE TITLE
UDP port is not possible to change

### ISSUE THREAD

**Issue by jjtoard** [hassio-addons/addon-wireguard#30]
I tried to change the UDP port wireguard is listening on but it looks to keep his default port 51820.
At least the log keeps reporting it is listening on 51820 so maybe it works correctly but didn't test it

**Comment by sinclairpaul** [comment]
Please test it and report an issue if needed, the log states the port used internally in the container.

**Comment by jjtoard** [comment]
it is the log that reports it is listening on port 51820 even that I changed the port an that it is also changed in the client config files.  So it is at least a reporting issue in the logs and maybe also in the configuration but didn't test that because I have to nat twice and my internetprovider doesn't like it if I change it a lot. (modem hangs after a few changes making it very difficult to troubleshoot)

**Comment by frenck** [comment]
The port is correct in the log. Inside the add-on, the WireGuard can run on a different port (as a matter of fact, internally it always runs on 51820).

Changing the port setting in Hass.io, changes the port that is facing outwards when routing out the traffic from the add-on. Which does not affect add-on internals, but is handled by Docker.

TL;DR. The port listing in the logs is correct and not a bug.
